### ISSUE TITLE
Powerpoint 2013: When running slide shows, NvDA does not announce contents of the slide being shown

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3602]
**Reported by nvdakor on 2013-10-23 00:45**
Hi,
This is a continuation from #3578:
After creating some slides, when you run slide shows 9F5), NvDA does not announce the contents of the slide shown on screen. To reproduce this, follow steps outlined in #3578 up to creating new slides. Then from the first slide, press F5 to run the slide show. You'll get:
Expected: When slide is shown, NVDA should announce the contents of the slide.
Actual: NvDA does not announce this info even when there's content in a given slide.
Haven't tested with speaker notes yet.
System specs:
- Windows 7 64-bit.
- Powerpoint 2013 64-bit.
- Version: latest next snapshot.
  Thanks.

**Comment by jcsteh** [comment]
@MichaelDCurran, is this fixed? Looks like it is except for protected view, but I'm not sure we can do anything about that?

**Comment by Adriani90** [comment]
In Powerpoint 2016 / 365 this is fixed. I am testing with NVDA 2019.1.1. I guess this is also the case for powerpoint 2013 since both rely on UIA as far as I know. I am closing as works for me.
@feerrenrut if you can reproduce this in powerpoint 2013, feel free to reopen it.
